esp/scm/ws_packageprocess.ecm

```diff
@@ -288,7 +288,7 @@ ESPresponse [exceptions_inline] GetPartFromPackageMapResponse
     string Content;
 };
 
-ESPservice [requiredaccess("NONE"), version("1.02"), exceptions_inline("./smc_xslt/exceptions.xslt")] WsPackageProcess
+ESPservice [autofeatureauth("NONE"), version("1.02"), exceptions_inline("./smc_xslt/exceptions.xslt")] WsPackageProcess
 {
```

Comment:
@wangkx @afishbeck it looks like this entire service lacks feature level auth, can you guys confirm this was done by design?